cts/cli/regression.crm_mon.exp

```diff
@@ -603,8 +603,11 @@ Cluster Summary:
   * 32 resource instances configured (4 DISABLED)
 
 Node List:
-  * Online: [ cluster01 (1) cluster02 (2) ]
-  * GuestOnline: [ httpd-bundle-0@cluster01 httpd-bundle-1@cluster02 ]
+  * Node cluster01 (1): online, feature set <3.15.1
+  * Node cluster02 (2): online, feature set <3.15.1
+  * GuestNode httpd-bundle-0@cluster01: online
+  * GuestNode httpd-bundle-1@cluster02: online
+  * GuestNode httpd-bundle-2@: OFFLINE
```

Comment:
Is the change for the GuestNode lines intentional?

Reply:
Sort of ... with --show-detail, if any of the nodes have to be listed individually, they all are. With this change, cluster nodes will always need to be listed individually, so in effect this change means that --show-detail now shows all nodes individually.